### PR TITLE
#619 Using table name in row checkbox labels

### PR DESCRIPTION
**Pull request by nmgokhale**
Fixes https://github.com/elyra-ai/canvas/issues/619

Reverted changes in https://github.com/elyra-ai/canvas/commit/302ead3ebcfcc672439993f128160ae60f40d328#diff-6a74ae667bc839b609875b48b841ab0a1270614d6857a11999f7ebde8bd3e6a2
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: canvas_modules/common-canvas/src/common-properties/components/flexible-table/flexible-table.jsx [hunk 1]
```diff
@@ -419,10 +420,13 @@ export default class FlexibleTable extends React.Component {
 		let searchBar = null;
 
 		if (typeof this.props.filterable !== "undefined" && this.props.filterable.length !== 0) {
-			const placeHolder = formatMessage(this.props.controller.getReactIntl(),
-				MESSAGE_KEYS.TABLE_SEARCH_PLACEHOLDER) + " " + searchLabel;
-			const searchBarLabel = formatMessage(this.props.controller.getReactIntl(),
-				MESSAGE_KEYS.TABLE_SEARCH_LABEL, { table_name: this.props.tableLabel });
+			const placeHolder = this.props.intl.formatMessage(
```
Comment (matthoward366) [inline canvas_modules/common-canvas/src/common-properties/components/flexible-table/flexible-table.jsx:423]: Are we concatenating labels here?

Comment (nmgokhale) [reply canvas_modules/common-canvas/src/common-properties/components/flexible-table/flexible-table.jsx:423]: Yes, `"table.search.placeholder": "Find in column",`
We show -` Find in column {Column_name}`

Comment (matthoward366) [reply canvas_modules/common-canvas/src/common-properties/components/flexible-table/flexible-table.jsx:423]: ok, this should be fix.

**Review by matthoward366 — APPROVED** [review]
Changes look good.

**Pull request merged**